assume/markets/clearing_algorithms/complex_clearing_dmas.py

```diff
@@ -27,7 +27,7 @@
 
 logger = logging.getLogger(__name__)
 
-SOLVERS = ["glpk", "cbc", "gurobi", "cplex"]
+SOLVERS = ["appsi_highs", "glpk", "cbc", "gurobi", "cplex"]
```

Comment:
maybe we should stick to the same order of solvers for now among files before we have a global variable for that?